src/composables/latestTransactionList.ts

```diff
@@ -102,18 +104,23 @@ export function useLatestTransactionList() {
 
   /**
    * Add temporary pending transaction and remove it when it's mined.
-   * For now this is usable only with Aeternity protocol
    */
   async function addAccountPendingTransaction(address: AccountAddress, transaction: ITransaction) {
     const account = getAccountByAddress(address);
-    if (account?.protocol === PROTOCOLS.aeternity && transaction?.hash) {
+    if (
+      account
+      && transaction?.hash
+      && includes([PROTOCOLS.aeternity, PROTOCOLS.ethereum], account?.protocol)
```

Comment:
It throws a TS issue because the array doesn't include `bitcoin` and `account?.protocol` could be `bitcoin` (which is why we're using `includes` so it doesn't make sense 🤷‍♂️)
I wanted to avoid the `as any` but perhaps it is preferable than using lodash

Reply:
Yeah, I see the issue. We have alreade replacement for `includes` method to comply with TS. Please take a look on `includes` utility in the `utils/common.ts` file.